src/tactic/replacer.lean

```diff
@@ -6,7 +6,7 @@ Author: Mario Carneiro
 A mechanism for defining tactics for use in auto params, whose
 meaning is defined incrementally through attributes.
 -/
```

Comment:
I think @digama0 is the author here.

Reply:
I think that's what is says at the moment, so I'll mark this as resolved.